opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/feign/FeignDefinedUrlTest.java

```diff
@@ -60,6 +61,11 @@
   @Autowired
   protected FeignInterface feignInterface;
 
+  @Before
```

Comment:
Could you please revert changes in the files which should not be subject to this PR.

Reply:
done